sklearn/utils/validation.py

```diff
@@ -498,10 +498,10 @@ def check_X_y(X, y, accept_sparse=None, dtype="numeric", order=None, copy=False,
                     ensure_2d, allow_nd, ensure_min_samples,
                     ensure_min_features, warn_on_dtype, estimator)
     if multi_output:
-        y = check_array(y, 'csr', force_all_finite=True, ensure_2d=False,
-                        dtype=None)
+        y = check_array(y, 'csr', copy=copy, force_all_finite=True,
+                        ensure_2d=False, dtype=None)
```

Comment:
Yes, I suppose we should pass these arguments as well.

For now, I think I should use `check_array` twice for both X and y as a workaround.